package.json

```
@@ -33,6 +33,7 @@
     "build:dapp": "yarn workspace @ubiquity/dapp run build",
     "dev:dapp": "yarn workspace @ubiquity/dapp run start",
     "test:all": "yarn workspace @ubiquity/contracts run test:unit",
+    "coverage": "cd packages/contracts/ && ./coverage.sh",
```

```suggestion
    "coverage": "yarn workspace @ubiquity/contracts run test:coverage",
```

Perhaps we should add `{ "test:coverage": "./test-coverage.sh" }` to `contracts/package.json`